solutions/1029/01.py

```
@@ -0,0 +1,17 @@
+class Solution:
+    def twoCitySchedCost(self, costs: List[List[int]]) -> int:
```

Ruff (0.14.7)

Undefined name `List`

(F821)